typings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "globalDependencies": {
+      "express": "github:DefinitelyTyped/DefinitelyTyped/express/express.d.ts#dd4626a4e23ce8d6d175e0fe8244a99771c8c3f2",
+      "serve-static": "github:DefinitelyTyped/DefinitelyTyped/serve-static/serve-static.d.ts#0fa4e9e61385646ea6a4cba2aef357353d2ce77f",
+      "mime": "github:DefinitelyTyped/DefinitelyTyped/mime/mime.d.ts#cb5206a8ac1c9a3ddfd126f5ecea6729b2361452",
+      "node": "github:DefinitelyTyped/DefinitelyTyped/node/node.d.ts#1c56e368e17bb28ca57577250624ca5bd561aa81",
+      "socket.io": "github:DefinitelyTyped/DefinitelyTyped/socket.io/socket.io.d.ts#4128f7af06c355e3526b7cf7cbb85e2e43bff167"
+  }
```

Comment:
Is not an automatic deps ? Should be moved in plugin
Le 7 juin 2016 10:36, "Nicola Rizzo" <notifications@github.com> a écrit :

> In typings.json
> <https://github.com/bap-node-microframework/node-microframework-ping-module/pull/1#discussion_r66030107>
> :
>
> > @@ -0,0 +1,9 @@
> > +{
> > +  "globalDependencies": {
> > +      "express": "github:DefinitelyTyped/DefinitelyTyped/express/express.d.ts#dd4626a4e23ce8d6d175e0fe8244a99771c8c3f2",
> > +      "serve-static": "github:DefinitelyTyped/DefinitelyTyped/serve-static/serve-static.d.ts#0fa4e9e61385646ea6a4cba2aef357353d2ce77f",
> > +      "mime": "github:DefinitelyTyped/DefinitelyTyped/mime/mime.d.ts#cb5206a8ac1c9a3ddfd126f5ecea6729b2361452",
> > +      "node": "github:DefinitelyTyped/DefinitelyTyped/node/node.d.ts#1c56e368e17bb28ca57577250624ca5bd561aa81",
> > +      "socket.io": "github:DefinitelyTyped/DefinitelyTyped/socket.io/socket.io.d.ts#4128f7af06c355e3526b7cf7cbb85e2e43bff167"
> > +  }
>
> Socket is a dependency of bap-node-microframework that is a dependency of
> ping module
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/bap-node-microframework/node-microframework-ping-module/pull/1/files/58f03cd5d99dde4093ea898ee0fe913aa880b2d6#r66030107>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnw3CtHjyJwJr4HAI9q_ToJ7o15b-Oqks5qJS2jgaJpZM4IvrNU>
> .
>